src/BuildServer/Services/ArtifactBuilders/ArtifactBuilder.cs

```diff
@@ -59,5 +64,19 @@ public bool Build(Build build)
                 process.Dispose();
             }
         }
+
+        private string ValidateParameter(string parameter)
+        {
+            Regex regex = new Regex(@"^[a-zA-Z0-9./\-_]*$");
+            Match match = regex.Match(parameter);
+            if (match.Success)
+            {
+                return parameter;
+            }
+
+            this.logger.LogError($"Invalid parameter. " +
```

Comment:
nit: what do you think about adding the provided parameter to this `LogError` call so we're more clear about what parameter we're referring to?

```cs
this.logger.LogError($"Invalid parameter provided: '{parameter}'. Only alpha-numeric, '/', '.', '-', '_' characters are allowed");
```

Reply:
Ah I meant to add this, thanks!